adoc/chapters/programming_interface.adoc

```diff
@@ -1420,11 +1420,19 @@ info::context::atomic_memory_order_capabilities
 ----
 
     @ [.code]#std::vector<memory_order>#
-   a@ Returns the set of memory orderings supported by atomic operations on all
-      devices in the context, which is guaranteed to include [code]#relaxed#.
+   a@ This query applies only to the capabilities of atomic operations that are
+      applied to memory that can be concurrently accessed by multiple devices
+      in the context.  If these capabilities are not uniform across all devices
+      in the context, the query reports only the capabilities that are common
+      for all devices.
 
-The memory ordering of the context determines the behavior of atomic operations applied
-to any memory that can be concurrently accessed by multiple devices in the context.
+Returns the set of memory orders supported by these atomic operations.  When a
+context returns a "stronger" memory order in this set, it must also return all
+"weaker" memory orders.  The memory orders [code]#memory_order::acquire#,
+[code]#memory_order::release#, and [code]#memory_order::acq_rel# are all the
+same strength.  If a context returns one of these, it must return them all.
+
+At a minimum, each context must support [code]#memory_order::relaxed#.
```

Comment:
We are following OpenCL here, which has the same concept of "strength":

> Because atomic memory orders are hierarchical, a device that supports a strong memory order must also support all weaker memory orders.

(https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#CL_DEVICE_ATOMIC_MEMORY_CAPABILITIES)